conference.js

```diff
@@ -71,7 +72,7 @@ const eventEmitter = new EventEmitter();
 let room;
 let connection;
 let localAudio, localVideo;
-let initialAudioMutedState = false, initialVideoMutedState = false;
+let initialAudioMutedState = false;
```

Comment:
The muted attribute we have in media is agnostic to the track, it will be synced to it. So if we have no track, we can set it there, and the moment a track appears it will be muted. Now, the problem is not everything on the web is using React / Redux, so this may not be the case for the web.